tests/formatter_tests.py

```diff
@@ -7,6 +7,7 @@
 from dateutil import tz as dateutil_tz
 
 from arrow import formatter
+from tests.utils import make_full_tz_list
```

Comment:
This works, but it may be better to say `from .utils import make_full_tz_list` since you are importing from the same directory.

Reply:
Done!